package.json

```diff
@@ -18,24 +18,32 @@
     "url": "https://github.com/stefanjudis/perf-tooling/issues"
   },
   "scripts": {
-    "test": "gulp test"
+    "test": "gulp test",
+    "gulp": "gulp",
+    "dev": "gulp dev",
+    "server": "nodemon --watch templates/ --watch public --watch app.js --ext tpl,css,js app.js"
   },
   "homepage": "http://perf-tooling.today",
   "dependencies": {
     "MD5": "^1.2.1",
     "async": "^0.9.0",
     "compression": "^1.1.0",
+    "cookie-parser": "^1.3.4",
     "express": "^4.9.0",
     "gulp": "~3.8.0",
     "gulp-autoprefixer": "~2.1.0",
     "gulp-combine-media-queries": "^0.2.0",
     "gulp-concat": "~2.4.1",
+    "gulp-connect": "^2.2.0",
```

Comment:
I think it was left by mistake. I think it was part of my try to set up a live reload configuration and it's not relevant now, because you already configured the nodemon.